### PR TITLE
DaemonSet pod priorityclass

### PR DESCRIPTION
**Pull request by cedricmckinnie**
Added the ability to specify a PriorityClass to DaemonSet pods. This can be important when using a nodeSelector to make sure that other pods running on those nodes don't preempt Keepalived pods and create unintended imbalances.

### CODE REVIEW

**Comment by cedricmckinnie** [comment]
@raffaelespazzoli Any way you think this could be simplified?

File: api/v1alpha1/zz_generated.deepcopy.go [hunk 1]
```diff
@@ -108,6 +108,13 @@ func (in *KeepalivedGroupSpec) DeepCopyInto(out *KeepalivedGroupSpec) {
 		*out = make([]int, len(*in))
 		copy(*out, *in)
 	}
+	if in.DaemonsetPodAnnotations != nil {
```
Comment (cedricmckinnie) [inline api/v1alpha1/zz_generated.deepcopy.go:111]: Artifact of a previous feature added. Seems like this can be ignored

**Comment by cedricmckinnie** [comment]
@raffaelespazzoli sorry to be a bother but any thoughts/updates on this?

**Comment by cedricmckinnie** [comment]
@raffaelespazzoli  Seeing some build errors preventing the release.

Build error appears to have been caused by a missing image:
https://github.com/redhat-cop/keepalived-operator/actions/runs/4358129606/jobs/7631527579#step:3:33

The image appears to have been successfully uploaded:
https://github.com/redhat-cop/keepalived-operator/actions/runs/4358129606/jobs/7631419807#step:15:38

Could it be related to the recent updates to https://github.com/redhat-cop/github-workflows-operators ?

**Comment by raffaelespazzoli** [comment]
I think upgrading the build pipeline to 1.0.5 should fix the issue. I need to test it.

**Comment by cedricmckinnie** [comment]
Would love to avoid having to fork the template into a ConfigMap for my current use case. Anything I can do to assist here? Many thanks!

**Comment by raffaelespazzoli** [comment]
This has been released, has it not?

**Comment by cedricmckinnie** [comment]
Ah indeed it is. Thanks and apologies!

**Pull request merged**